spec/CLI.spec.js

```diff
@@ -267,70 +271,79 @@ describe('execution', () => {
+    async done => {
+      const env = { ...process.env };
+      env.NODE_OPTIONS = '--dns-result-order=ipv4first --trace-deprecation';
+      childProcess = spawn(
+        binPath,
+        [
+          '--appId',
+          'test',
+          '--masterKey',
+          'test',
+          '--databaseURI',
+          databaseURI,
+          '--port',
+          '1340',
+          '--mountGraphQL',
+        ],
+        { env }
+      );
+      handleStdout(childProcess, done, aggregatedData, [
+        'parse-server running on',
+        'GraphQL running on',
+      ]);
+      handleStderr(childProcess, done);
+      handleError(childProcess, done);
+    }
+  );
 
-  it_id('2769cdb4-ce8a-484d-8a91-635b5894ba7e')(it)('should start Parse Server with GraphQL and Playground', async done => {
-    const env = { ...process.env };
-    env.NODE_OPTIONS = '--dns-result-order=ipv4first --trace-deprecation';
-    childProcess = spawn(
-      binPath,
-      [
-        '--appId',
-        'test',
-        '--masterKey',
-        'test',
-        '--databaseURI',
-        databaseURI,
-        '--port',
-        '1341',
-        '--mountGraphQL',
-        '--mountPlayground',
-      ],
-      { env }
-    );
-    handleStdout(childProcess, done, aggregatedData, [
-      'parse-server running on',
-      'Playground running on',
-      'GraphQL running on',
-    ]);
-    handleStderr(childProcess, done);
-    handleError(childProcess, done);
-  });
+  it_id('2769cdb4-ce8a-484d-8a91-635b5894ba7e')(it)(
+    'should start Parse Server with GraphQL and Playground',
+    async done => {
+      const env = { ...process.env };
+      env.NODE_OPTIONS = '--dns-result-order=ipv4first --trace-deprecation';
+      childProcess = spawn(
+        binPath,
+        [
+          '--appId',
+          'test',
+          '--masterKey',
+          'test',
+          '--databaseURI',
+          databaseURI,
+          '--port',
+          '1341',
+          '--mountGraphQL',
+          '--mountPlayground',
+        ],
+        { env }
+      );
+      handleStdout(childProcess, done, aggregatedData, [
+        'parse-server running on',
+        'Playground running on',
+        'GraphQL running on',
+      ]);
+      handleStderr(childProcess, done);
+      handleError(childProcess, done);
+    }
+  );
 
-  it_id('23caddd7-bfea-4869-8bd4-0f2cd283c8bd')(it)('can start Parse Server with auth via CLI', done => {
-    const env = { ...process.env };
-    env.NODE_OPTIONS = '--dns-result-order=ipv4first --trace-deprecation';
-    childProcess = spawn(
-      binPath,
-      ['--databaseURI', databaseURI, './spec/configs/CLIConfigAuth.json'],
-      { env }
-    );
-    handleStdout(childProcess, done, aggregatedData, ['parse-server running on']);
-    handleStderr(childProcess, done);
-    handleError(childProcess, done);
-  });
+  it_id('23caddd7-bfea-4869-8bd4-0f2cd283c8bd')(it)(
+    'can start Parse Server with auth via CLI',
+    done => {
```

Comment:
What line length is this enforcing? Looks less readable to me this way.

Reply:
Looks like prettier has a 100 line length

Reply:
Should we set this to longer lines, like 170?

Reply:
We could but it’s up to you to review 170+ files being changed. My plan was to remove prettier and husky anyway like the JS SDK. Should I just redo this PR without running prettier?